src/cryptonote_protocol/levin_notify.cpp

```diff
@@ -286,7 +286,6 @@ namespace levin
           connection_count(0),
           flush_callbacks(0),
           nzone(zone),
-          is_public(is_public),
```

Comment:
Looks like this got added in a merge error due to dff1d8067c38ecebce9b5d7a7a3d5edc54669b33, probably wasn't rebased correctly.